src/test/java/seedu/address/model/person/OrganisationTest.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class OrganisationTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Organisation(null));
+    }
+
+    @Test
+    public void constructor_invalidOrganisation_throwsIllegalArgumentException() {
+        String invalidOrganisation = "";
+        assertThrows(IllegalArgumentException.class, () -> new Organisation(invalidOrganisation));
+    }
+
+    @Test
+    public void isValidOrganisation() {
+        // null address
```

Comment:
I think you mean "null organisation".

Reply:
yes, I meant "null organisation". Good find.